### ISSUE TITLE
volpiano sometimes renders weirdly?

### ISSUE THREAD

**Issue by annamorphism** [DDMAL/CantusDB#1727]
A user reports that high F looks strange in his Volpiano. 
The chant in question looks fine to me: https://cantusdatabase.org/chant/680985 (Volpiano is only in the edit view.)
But here is his screenshot (using Bing):

![image](https://github.com/user-attachments/assets/bafdb8af-ea20-404b-8313-da79c50613b2)

**Comment by dchiller** [comment]
Hmmm...it looks fine to me in Chrome, Firefox, and Safari so I suspect this is a browser issue. I also don't know what it even means that the user is screenshotting in Bing? That's a search engine?

Perhaps if we have more information we can look into this. Does it stay this way even on hard reloads of the page (shift + reload)?

**Comment by annamorphism** [comment]
....yes, as I copied out his email quickly I thought "Bing? that's not a browser?"--presumably we're dealing with Edge. 
I'll ask about shift+reload.

**Comment by annamorphism** [comment]
I asked about hard reset. 
"I did "crtl + shift + R", but nothing changed..."

**Comment by dchiller** [comment]
ok, I am downloading edge to see if I can reproduce.

**Comment by dchiller** [comment]
Hmm...this is what I see on Edge:

![image](https://github.com/user-attachments/assets/a4dfe33e-7eb5-47ff-a73d-a172bdee57d4)

What happens if they try on a different browser? Or update Edge? 

Other than that, I don't really know what I can do...

**Comment by annamorphism** [comment]
well, I think we are satisfied that it isn't giving a weird result to every Windows user or something. I will try to suss out more about this problem if it continues to arise, but we can table it for now.

**Comment by dchiller** [comment]
Hmm.... @kyrieb-ekat that's not the volpiano font though... is that melody encoded a la volpiano and then rendered with some other font or through some other software?

**Comment by kyrieb-ekat** [comment]
Yes- that was my bad and why I deleted (hope it's showing as deleted??), I don't think the process would apply to what the user did; cantus is pretty good at wrapping/moving stuff around based on zoom and window changes.

**Comment by dchiller** [comment]
Ah yes, now deleted :) I had these page open in a tab, so maybe it refreshed when you added the comment but then didn't refresh when you deleted it -- All good.

**Comment by dchiller** [comment]
Going to close this for now since it hasn't come back up and we feel pretty certain that this wasn't anything on our end.

**Comment by annamorphism** [comment]
This problem is back!
Screenshots from an email yesterday:
"I'm using a dell (latitude, I think?) and Firefox on Windows 10.
I've just checked it on Microsoft Edge and Chrome, and the issue persists in them too."

![Image](https://github.com/user-attachments/assets/68b8b563-818c-4c42-bef2-87d36b39683b)
![Image](https://github.com/user-attachments/assets/081d9323-e8b0-455d-82db-6449286221e6)

**Comment by dchiller** [comment]
@annamorphism Just to clarify that in your screenshots there are two slightly different problems, right? The first circled bit doesn't have a high f, but in that case it's a mid-staff flat that's showing weirdly...

**Comment by annamorphism** [comment]
Right, there's the half-rendered F as well as the weird rectangles. The original complaint had both, too, but it didn't seem to be associated with a flat in that case...

**Comment by dchiller** [comment]
How well do we know them? Can I ask them to do some tests on their computer? 

I think the goal would be to isolate if this is actually something CantusDB related as opposed to *just* a volpiano compatibility issue where there's nothing we can do about it.

**Comment by annamorphism** [comment]
I don't know them well but they volunteered they were "happy to keep troubleshooting" (and gave the OS/browser information without prompting) so they probably could run some tests.
My gut feeling is that it's some sort of Volpiano/Windows compatibility thing, but, would be nice to test...

**Comment by dchiller** [comment]
Ok, what I would do in that case is send you/them an html file via email, ask them to download it and open it in their browser and send us a screenshot.

**Comment by annamorphism** [comment]
as an aside, it's interesting that this most recent user sees it in both Volpiano windows, where the other only has it in one but not the "preview with text": 

![Image](https://github.com/user-attachments/assets/3461a475-8d92-4cbd-b5b7-ee1ae0c82328)

**Comment by kyrieb-ekat** [comment]
Cannot reproduce

**Comment by annamorphism** [comment]
on hold/wontfix unless/until we manage to reproduce this
